doc/operations/recovery.rst

```diff
@@ -145,6 +145,94 @@ Which of these two paths is taken is noted in the `public:ccf.internal.last_reco
       ...
     $ /opt/ccf/bin/js_generic --config /path/to/config/file
 
+Self-Healing-Open recovery
+--------------------------
+
+In environments with limited orchestration or limited operator access, it is desirable to allow an automated disaster recovery without operator intervention.
+At a high level, Self-Healing-Open recovery allows recovering replicas to discover which node has the most up-to-date ledger and automatically recover the network using that ledger.
```

Comment:
And explanation early on that this is only useful when post-DR nodes are guaranteed to restart on at least some of the same hardware, i.e. NOT in ACI.